engine/org.eclipse.birt.report.engine.emitter.pptx.tests/src/org/eclipse/birt/report/engine/emitter/pptx/tests/TestPPTX2PNG.java

```diff
@@ -29,6 +30,7 @@ public void testText() throws Exception {
 	}
 
 	@Test
+	@Ignore
```

Comment:
@speckyspooky 

I ignored this test because it gets an NPE.  But I don't know enough about report design to fix that.

I would highly recommend removing the `@Ignore`  and run the test in the IDE to see the failure:

![image](https://github.com/eclipse-birt/birt/assets/208716/dddf121a-25f1-4524-9ea0-2a1dcbacd6a6)

I think the report sample needs a non-null table style but I have no idea what that means....